### PR TITLE
sample: charger

### PR DESCRIPTION
**Pull request by wilu-nordic**

### CODE REVIEW

File: samples/charger_and_events/Kconfig [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+mainmenu "PMIC example: charger and events"
```
Comment (masz-nordic) [inline samples/charger_and_events/Kconfig:7]:
```suggestion
mainmenu "Charger and events sanoke"
```

File: samples/charger_and_events/Kconfig [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+config BATTERY_VOLTAGE_THRESHOLD_1
+	int "Battery voltage threshold 1 [mV]."
```
Comment (masz-nordic) [inline samples/charger_and_events/Kconfig:10]:
```suggestion
	int "Battery voltage threshold 1 (in millivolts)."
```
Same below.

File: samples/charger_and_events/README.rst [hunk 3]
```diff
@@ -0,0 +1,108 @@
+.. _npmx_charger_and_events_sample:
+
+npmx: CHARGER and EVENTS
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The npmx charger and events sample demonstrates how to control PMIC charger and receive events using npmx drivers.
```
Comment (masz-nordic) [inline samples/charger_and_events/README.rst:10]:
```suggestion
The charger and events sample demonstrates how to control charger and receive events using npmx drivers.
```

File: samples/charger_and_events/README.rst [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+Check and configure the following sample-specific Kconfig options:
+
```
Comment (masz-nordic) [inline samples/charger_and_events/README.rst:78]: What about `BATTERY_VOLTAGE_THRESHOLD_1` and `_2`?

File: samples/charger_and_events/sample.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+sample:
+  description: Sample for nPM1300 PMIC Charger and events
+  name: nPM1300 PMIC Charger and events
```
Comment (masz-nordic) [inline samples/charger_and_events/sample.yaml:3]:
```suggestion
  description: Charger and events sample
  name: Charger and events
```

File: samples/charger_and_events/README.rst (hunk 4 above)
Comment (wilu-nordic) [reply samples/charger_and_events/README.rst:78]: Added.

File: samples/charger_and_events/Kconfig [hunk 6]
```diff
@@ -0,0 +1,81 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+mainmenu "Charger and events sample"
```
Comment (annakielar) [inline samples/charger_and_events/Kconfig:7]:
```suggestion
mainmenu "CHARGER and EVENTS sample"
```
@masz-nordic @wilu-nordic When talking about a sample, should we capitalize CHARGER and EVENTS? If so, please correct all occurrences.

File: samples/charger_and_events/README.rst [hunk 7]
```diff
@@ -0,0 +1,118 @@
+.. _npmx_charger_and_events_sample:
+
+npmx: CHARGER and EVENTS
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The charger and events sample demonstrates how to control charger and receive events using npmx drivers.
```
Comment (annakielar) [inline samples/charger_and_events/README.rst:10]: As that is the intro sentence, can we be more specific about the charger? Do you mean any specific charger, e.g., for any specific device or a type of device?

File: samples/charger_and_events/README.rst [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, connect and disconnect battery and USB-C cable to see various event messages.
+Battery charging status will also be printed.
+
+#. |connect_kit|
+#. |connect_terminal|
```
Comment (annakielar) [inline samples/charger_and_events/README.rst:113]:
```suggestion
After programming the sample to your development kit, do the following:

#. Connect and disconnect the battery and the USB-C cable to see various event messages.
   Battery charging status will also be printed.
#. |connect_kit|
#. |connect_terminal|
```

**Review by annakielar — CHANGES_REQUESTED** [review]

File: samples/charger_and_events/Kconfig (hunk 6 above)
Comment (masz-nordic) [reply samples/charger_and_events/Kconfig:7]: Both versions are ok to me.

File: samples/charger_and_events/README.rst [hunk 9]
```diff
@@ -0,0 +1,118 @@
+.. _npmx_charger_and_events_sample:
+
+npmx: CHARGER and EVENTS
```
Comment (masz-nordic) [inline samples/charger_and_events/README.rst:3]:
```suggestion
CHARGER and EVENTS
```

File: samples/charger_and_events/README.rst [hunk 10]
```diff
@@ -0,0 +1,118 @@
+.. _npmx_charger_and_events_sample:
```
Comment (masz-nordic) [inline samples/charger_and_events/README.rst:1]:
```suggestion
.. _charger_and_events_sample:
```

File: samples/charger_and_events/README.rst (hunk 3 above)
Comment (annakielar) [reply samples/charger_and_events/README.rst]: @masz-nordic @wilu-nordic 
Marcin, can we keep "PMIC" acronym just for clarity? As this is a very first intro sentence, would be great if we can be as specific as possible and use proper names at least at the beginning of this page.

File: samples/charger_and_events/README.rst [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, do the following:
+
+#. Connect and disconnect the battery and the USB-C cable to see various event messages. Battery charging status will also be printed.
+#. |connect_kit|
+#. |connect_terminal|
```
Comment (annakielar) [inline samples/charger_and_events/README.rst:113]:
```suggestion
1. |connect_kit|
#. |connect_terminal|
#. Connect and disconnect the battery and the USB-C cable to see various event messages. Battery charging status will also be printed.
```

**Review by annakielar — CHANGES_REQUESTED** [review]

File: samples/charger_and_events/README.rst [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+********************
+
+.. |sample path| replace:: :file:`samples/charger_and_events`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, do the following:
```
Comment (annakielar) [inline samples/charger_and_events/README.rst:109]:
```suggestion
|test_sample|
```

**Review by annakielar — CHANGES_REQUESTED** [review]

File: samples/charger_and_events/README.rst (hunk 3 above)
Comment (masz-nordic) [reply samples/charger_and_events/README.rst]: Okay

File: samples/charger_and_events/README.rst [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+This sample uses changer and events drivers to do the following:
```
Comment (annakielar) [inline samples/charger_and_events/README.rst:24]:
```suggestion
This sample uses CHARGER and Events drivers to do the following:
```

File: samples/charger_and_events/README.rst [hunk 14]
```diff
@@ -0,0 +1,118 @@
+.. _charger_and_events_sample:
+
+CHARGER and EVENTS
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The charger and events sample demonstrates how to control PMIC charger and receive events using npmx drivers.
```
Comment (annakielar) [inline samples/charger_and_events/README.rst:10]:
```suggestion
The CHARGER and Events sample demonstrates how to control a PMIC charger and receive events using npmx drivers.
```

**Review by annakielar — CHANGES_REQUESTED** [review]

**Review by annakielar — APPROVED** [review]

File: samples/charger_and_events/sample.yaml [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+
+common:
+  integration_platforms:
+    - nrf5340dk_nrf5340_cpuapp
+    - nrf52840dk_nrf52840
+  platform_allow: nrf5340dk_nrf5340_cpuapp nrf52840dk_nrf52840
+  tags: pmic
+
+tests:
+  sample.charger_and_events_eng_b:
```
Comment (masz-nordic) [inline samples/charger_and_events/sample.yaml:13]:
```suggestion
  sample.charger_and_events.eng_b:
```

**Review by annakielar — APPROVED** [review]

**Pull request merged**